data/human/events.txt

```diff
@@ -1392,6 +1392,9 @@ event "liberation of Poisonwood"
 		fleet "Small Free Worlds" 700
 		fleet "Small Free Worlds" 1200
 		fleet "Human Miners" 2000
+	planet Poisonwood
+		spaceport `The starport is a hodgepodge of small metal structures, built from the remains of Navy and pirate ships that fought for control of this world. The locals here welcome you to their planet heartily.`
+		spaceport `	The tavern in the center of the port is named The Parting Glass. The walls are covered with photographs from after the war: liberated slaves boarding Navy ships to leave this planet that had brought them so much pain; Navy officers leaving their ships to retire on this world that they liberated; friends gone away, and friends left behind. There are also newer photos mixed in amongst the older ones: two Falcons and a Navy Cruiser landed side by side; troops clad in armor walking away from a factory, and a familiar captain talking to a man you know as Commander Nguyen. It is a strange feeling to be shown along heroes of a history that you have had only the smallest effect on.`
```

Comment:
```suggestion
		spaceport `	The tavern in the center of the port is named The Parting Glass. Its walls are covered with photographs from after the war: liberated slaves boarding Navy ships to leave this planet that had brought them so much pain; Navy officers leaving their ships to retire on this world that they liberated; friends gone away; friends left behind. There are also newer photos mixed in amongst the older ones: two Falcons and a Navy Cruiser landed side by side; troops clad in armor walking away from a factory, and a familiar captain talking to a man you know as Commander Nguyen. It is a strange feeling to be shown along heroes of a history that you have had only the smallest effect on.`
```

(too many uses of the word "the" in close proximity)

Reply:
Should also probably be "shown _among_ heroes".

Reply:
Indeed, that too.